lib/numeric.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import numeral from 'numeral'
+
```

Comment:
ah, yeah. That's great. I'd hand-rolled some kind of formatting thing in the map repo too